src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
```

Comment:
```suggestion
public class Deadline extends Task {
```
Whitespace should be followed after class name.

Reply:
Also missing header comment for public class. If not LGTM, just some nits to fix.